localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ fire_cult:0 "Ragnarism"
+ fire_cult_adj:0 "Ragnaric"
+ fire_cult_adherent:0 "Ragnarist"
+ fire_cult_adherent_plural:0 "Ragnarists"
+ fire_cult_desc:0 "$fire_cult_adherent_plural$ worship one of the Elemental Lords, Ragnaros the Firelord, as their main god. Other fire elementals are seen as minor deities."
+ 
+ ordos_cult:0 "Ordosism"
+ ordos_cult_adj:0 "Ordosic"
+ ordos_cult_adherent:0 "Ordosist"
+ ordos_cult_adherent_plural:0 "Ordosists"
+ ordos_cult_desc:0 "Ordos was once a shaman of the yaungol who sacrificed himself to the flame, following the promises of the Fire Lords, in order to become a demigod. $ordos_cult_adj$ fire-keepers ritualistically set themselves ablaze to emulate the ascension of Ordos, and cast enchantments and prayers to ease his pain."
+ 
+ light_group:0 "Light"
+ light_group_adj:0 "Light"
+ light_group_adherent:0 "Light Follower"
+ light_group_adherent_plural:0 "Light Followers"
+ light_group_desc:0 "A group of religions worshipping a force known as the Light. They say its positive energies, residing in every living being, in every heart and soul, is everywhere, binding all sentient beings together as one. Those who reject this light inside are nothing more than heretics."
+ 
+ holy_light:0 "Holy Light"
+ holy_light_adj:0 "Holy Light"
+ holy_light_adherent:0 "Light Follower"
+ holy_light_adherent_plural:0 "Light Followers"
+ holy_light_desc:0 "The main organisation of the Holy Light has been the Church of the Holy Light. Most humans, dwarves and even some high elves follow the path of the Church. It is a faith that advocates purity, goodness, kindness, and compassion in the face of adversity."
+ 
+ scarlet_light:0 "Scarlet Light"
+ scarlet_light_adj:0 "Scarlet Light"
+ scarlet_light_adherent:0 "Scarlet Follower"
+ scarlet_light_adherent_plural:0 "Scarlet Followers"
+ scarlet_light_desc:0 "A heretical, militant offshoot of the Church of the Holy Light. They are intolerant of all other confessions, regardless of alliance or affiliation. They believe that any and all unholy creatures must be destroyed by a cleansing flame."
+ 
+ naaru:0 "Naarism"
+ naaru_adj:0 "Naaric"
+ naaru_adherent:0 "Naarist"
+ naaru_adherent_plural:0 "Naarists"
+ naaru_desc:0 "The Light manifests itself as a golden, shining light energy. The naaru, enigmatic beings composed of shards of fractured light taught the use of the Light to the draenei and other enlightened races, but others have come into the study and practice of Light-based magic on their own accord."
+ 
+ lightbound:0 "Lightbound"
+ lightbound_adj:0 "Lightbound"
+ lightbound_adherent:0 "Lightbound"
+ lightbound_adherent_plural:0 "Lightbound"
+ lightbound_desc:0 "A heretical offshoot of the naaru teachings, the adherents to the Lightbound doctrine are dedicated to spread the influence of the Light wherever they possibly can and convert the unbelievers by any means necessary, even forcibly."
```

Changed forcibly to through force, and adherents to to adherents of
```suggestion
 lightbound_desc:0 "A heretical offshoot of the naaru teachings, the adherents of the Lightbound doctrine are dedicated to spread the influence of the Light wherever they possibly can and convert the unbelievers by any means necessary, even through force."
```